test/compress/evaluate.js

```diff
@@ -95,6 +101,12 @@ unsafe: {
             2
         );
         console.log(
+            ({a:{b:1},a:1}) + 1,
+            2,
+            ({a:{b:1},a:1}).b + 1,
+            1..b + 1
```

Comment:
The test names I gave are just examples - please use whatever test names you think are appropriate.